vector/src/main/java/im/vector/app/features/userdirectory/UserDirectoryUserItem.kt

```diff
@@ -40,15 +43,23 @@ abstract class UserDirectoryUserItem : VectorEpoxyModel<UserDirectoryUserItem.Ho
     override fun bind(holder: Holder) {
         super.bind(holder)
         holder.view.setOnClickListener(clickListener)
-        // If name is empty, use userId as name and force it being centered
-        if (matrixItem.displayName.isNullOrEmpty()) {
-            holder.userIdView.visibility = View.GONE
-            holder.nameView.text = matrixItem.id
-        } else {
-            holder.userIdView.visibility = View.VISIBLE
-            holder.nameView.text = matrixItem.displayName
-            holder.userIdView.text = matrixItem.id
+        val displayName = matrixItem.displayName
```

Comment:
Check here if the `displayName` is missing
In order to build a default one with
`displayName = TchapUtils.computeDisplayNameFromUserId(matrixItem.id)`